repos/system_upgrade/el7toel8/actors/localreposinhibit/actor.py

```diff
@@ -0,0 +1,86 @@
+from leapp import reporting
+from leapp.actors import Actor
+from leapp.models import TargetRepositories, UsedTargetRepositories
+from leapp.reporting import Report
+from leapp.tags import IPUWorkflowTag, TargetTransactionChecksPhaseTag
+
+
+class LocalReposInhibit(Actor):
+    """Inhibits the upgrade if local repositories were found.
+
+    Currently we're not supporting local repositories during the upgrade.
```

Comment:
@pirat89 I would like to make sure you checked this docstrings, because it is describes why only custom repos to be checked

Reply:
Yes, this is nothing against I wrote above. I am focused around custom repositories only as well.